src/components/Account/Login/Login.actions.js

```diff
@@ -22,6 +23,61 @@ export function logout() {
 }
 
 export function login({ email, password }, type = 'local') {
+  const setAVoice = ({ loginData, dispatch, getState }) => {
+    const {
+      speech: {
+        voices,
+        options: { lang: voiceLang, voiceURI: browserVoiceUri }
+      }
+    } = getState(); //ATENTION speech options on DB is under Speech directly. on state is under options
+
+    const loginLanguage = loginData.settings?.language?.lang;
+    const userLanguage = loginLanguage?.substring(0, 2);
+    const browserVoiceLanguage = voiceLang?.substring(0, 2);
+
+    if (voices) {
+      const uris = voices.map(v => {
+        return v.voiceURI;
+      });
+      //if redux state have a defined voiceUri. Set it By default
+      if (
+        browserVoiceUri &&
+        browserVoiceLanguage === userLanguage &&
```

Comment:
This comparison is not a good idea. Most users are created in `en-US` and we never change that.

Reply:
The variable `userLanguage `  refers to the language.lang state. Should be renamed to `appLanguage`.